### PR TITLE
Added new `linkButton` to component demo

### PR DESCRIPTION
**Pull request by jamowei**
Needs https://github.com/jwstegemann/fritz2/pull/380 merged before

### CODE REVIEW

File: src/jsMain/kotlin/dev.fritz2.kitchensink/demos/buttonDemo.kt [hunk 1]
```diff
@@ -362,5 +363,103 @@ fun RenderContext.buttonDemo(): Div {
                 """
             )
         }
+
+        showcaseSection("Link")
+        paragraph {
+            +"There is also a special button for links which uses an "
```
Comment (haukesomm) [inline src/jsMain/kotlin/dev.fritz2.kitchensink/demos/buttonDemo.kt:369]: Minor wording suggestion: "There also is a special button [...]" over "There is also a special button [...]"

**Review by haukesomm — APPROVED** [review]

File: src/jsMain/kotlin/dev.fritz2.kitchensink/demos/buttonDemo.kt (hunk 1 above)
Comment (haukesomm) [reply src/jsMain/kotlin/dev.fritz2.kitchensink/demos/buttonDemo.kt:369]: Looks fine otherwise 👍🏻

**Review by WestHuus — DISMISSED** [review]
Looks good.

**Review by WestHuus — APPROVED** [review]
nice

**Pull request merged**